boot/bootutil/src/loader.c

```diff
@@ -1280,8 +1295,8 @@ boot_go(struct boot_rsp *rsp)
 
     case BOOT_SWAP_TYPE_PANIC:
     default:
-        /* TODO: what to do it a fatal error like flash read/write error
-         *       happened?
+        /* FIXME: what to do it a fatal error like flash read/write error
```

Comment:
How about logging an error and spinning forever? It seems better than jumping into potentially compromised code.